alembic/versions/e7040e76728_create_new_tables_for_quest.py

```diff
@@ -10,27 +10,37 @@
 revision = 'e7040e76728'
 down_revision = '508367dcbbb5'
 
+import datetime
+
 from alembic import op
 import sqlalchemy as sa
 
+def generate_default_id(context):
+    return context.current_parameters['name'].lower().replace(' ', '-')
 
 def upgrade():
 
     op.create_table(
         'team',
-        sa.Column('id', sa.Unicode(length=128), nullable=False),
+        sa.Column('id', sa.Unicode(length=128), nullable=False,
+                  default=generate_default_id),
```

Comment:
I have added this [method](http://docs.sqlalchemy.org/en/latest/core/defaults.html#context-sensitive-default-functions) in the migration file itself. It slugifies the input based on the name.